models/engine/db_storage.py

```diff
@@ -74,3 +74,43 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Get a single object from the database
+
+        Args:
+            cls     - string representing the class name
+            id      - string representing the object ID
+
+        Returns:
+            Object base on the class and id or else None.
+        """
+        if not cls or not id:
```

Comment:
The conditionals are not necessary here, type checking can be condensed in this method using an `eval` on the received class.